CHANGELOG.md

```diff
@@ -1,3 +1,268 @@
+| "/sync/resolve"                          | GET                 | DSRef            | remote.RefsHTTPHandler           |
+| "/sync/resolve"                          | DELETE              | --               | remote.RefsHTTPHandler           |
+|                                          |                     |                  |                                  |
+
+
+## Redesigned lib interface
+In general, we've streamlined the core functionality and reconciled input params in the `lib` package (which contain the methods and params that power both the api and cmd), so that no matter how you are accessing functionality in qri, whether using `lib` as a package,  using the api, or using the command line, you will have, not only consistent input parameters, but also consistent behavior. We're also utilizing our new `dispatch` pattern to replace our old rpc client with the same http json api that we expose to our users. That way all our api's, http or otherwise, have the same expectations. If something is broke in one place, it is broken in all places, consequently, when it is fixed in one place it will be fixed in all places.
+
+These changes will also help us in our upcoming challenges to refine expand the notion of identity inside of Qri.
+
+## Step-Based Transform Execution
+We've changed the underlying way that transforms execute, dividing the work into steps, like `Setup`, `Download`, and `Transform`. With this change, we are able to report the status of their transform back to the user. It will also give the user a better sense of what is happening in their transform: how long did the `Download` step take, which step failed, which steps worked. It also paves the way for a better transform development environment in the future. Users will be able to re-run certain steps or cache steps that they don't want to waste time re-running.
+
+## Other features:
+### Change report over API
+We've added an endpoint to get the change report via the api: `/changes`. This lays the ground work for powering any future UI we have surrounding change reports in our frontend applications.
+
+### Access and Oauth
+In preparation for multi tenency, we are adding support for generating JSON web tokens that will help ensure identity on the network. You can generate an access token via the cmd using the `qri access` command, or over the api using the `/access/token` endpoint.
+
+### Stats Component!
+We've added a new component: Stats! Stats is a component that contains statistical metadata about the body of a dataset. Its purpose is to provide an "at a glance" summary of a dataset. In order to remain fast for very large dataset sizes, we have opted to calculate the stats using probabilistic structures. Stats are an important part of change reports, and allow you to get a sense of what is different in a dataset body without having to examine that body line by line. 
```

Comment:
From a user's perspective this is actually a very big deal. I'd move this up to just after the `Resdesigned Lib Interface` section.

```suggestion
### Stats Component!
We've added a new component: Stats! Stats is a component that contains statistical metadata about the body of a dataset. Stats are now automatically calculated and saved with each new version.

Its purpose is to provide an "at a glance" summary of a dataset, calculating statistics on columns (ok, on "compound data types", but it's much easier to think about column stats). In order to remain fast for very large dataset sizes, we have opted to calculate the stats using probabilistic structures. Stats are an important part of change reports, and allow you to get a sense of what is different in a dataset body without having to examine that body line by line. 

For earlier versions that don't have stats calculated, we've added a `qri stats` command that will calculate the new stats component for you.
```